pkg/apply/applydrivers/apply_drivers_default.go

```diff
@@ -123,22 +122,15 @@ func (c *Applier) initStatus() {
 // todo: set up signal handler
 // todo(lingdie): use appErr to generate cmdPhase for each cmd and maintain a error map for images
 func (c *Applier) updateStatus(clusterErr error, appErr error) {
-	condition := v2.ClusterCondition{
-		Type:              "ApplyClusterSuccess",
-		Status:            v1.ConditionTrue,
-		LastHeartbeatTime: metav1.Now(),
-		Reason:            "Ready",
-		Message:           "Applied to cluster successfully",
-	}
-	c.ClusterDesired.Status.Phase = v2.ClusterSuccess
-	if clusterErr != nil {
-		condition.Status = v1.ConditionFalse
-		condition.Reason = "ApplyClusterError"
-		condition.Message = clusterErr.Error()
```

Comment:
should I change condition type if clusterErr is not nil?

Reply:
> should I change condition type if clusterErr is not nil?

yes.

Reply:
ok